delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -159,10 +159,9 @@ public void initialize(WorkerContext context) throws Exception {
 
     Path path = new Path(offsetBasePath);
     StateStore stateStore = StateStore.from(path);
-    EventMetrics eventMetrics = new EventMetrics(metrics, targetName);
     PipelineStateService stateService = new PipelineStateService(id, stateStore);
     stateService.load();
-    deltaContext = new DeltaContext(id, context.getRunId().getId(), metrics, stateStore, context, eventMetrics,
+    deltaContext = new DeltaContext(id, context.getRunId().getId(), metrics, stateStore, context,
```

Comment:
so we drop the information of targetName ? (the prefix)

Reply:
`targetName` in  metrics for dml and ddl events seems redundant. If we want we can emit separate metric for counting number of records inserted in the target, however there is no use case for that currently.